.appveyor.yml

```diff
@@ -32,62 +32,51 @@ environment:
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
       GENERATOR: "Visual Studio 14 2015 Win64"
       SDK: WpdPack
-      AIRPCAP: -DDISABLE_AIRPCAP=YES
```

Comment:
The number of test cases here should be reduced, with all tests not done with `-DDISABLE_AIRPCAP=YES` removed.